jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-client/src/test/java/org/jbpm/console/ng/es/client/editors/requestlist/RequestListViewTest.java

```diff
@@ -80,6 +80,14 @@ public void setDefaultFilterTitleAndDescriptionTest() {
 
         verify(filterPagedTableMock, times(7)).getMultiGridPreferencesStore();
         verify(filterPagedTableMock, times(7)).saveTabSettings(anyString(), any(HashMap.class));
+        verify(filterPagedTableMock).saveTabSettings(eq(RequestListViewImpl.REQUEST_LIST_PREFIX + "_0"), any(HashMap.class));
```

Comment:
@nmirasch would be useful if we also check that these methods are called in the expected order as well?
http://site.mockito.org/mockito/docs/current/org/mockito/Mockito.html#6